articles/faq/stay-on-plan-cost-more.md

```diff
@@ -9,4 +9,11 @@ For example if you are on a 1M plan and have the following sending pattern for t
 * Aug 19 to Sep 18: 1,136,456
 * Jul 20 to Aug 18: 1,232,012
 * Jun 20 to Jul 19: 986,472
+
 We would set your max monthly limit to 1,232,012 as this is the larger value than the plan value.
+
+
+** Please note **
```

Comment:
If you remove the spaces between the words and the asterisks, the text will be bolded: `**Please note**` results in **Please note**